test/vm.cpp

```diff
@@ -513,13 +513,39 @@ BOOST_AUTO_TEST_CASE(vmSystemOperationsTest)
 	dev::test::executeTests("vmSystemOperationsTest", "/VMTests", dev::test::doVMTests);
 }
 
+BOOST_AUTO_TEST_CASE(vmPerformanceTest)
+{
+	for (int i = 1; i < boost::unit_test::framework::master_test_suite().argc; ++i)
+	{
+		string arg = boost::unit_test::framework::master_test_suite().argv[i];
+		if (arg == "--performance")
+		{
+			auto start = chrono::steady_clock::now();
+
+			dev::test::executeTests("vmPerformanceTest", "/VMTests", dev::test::doVMTests);
+
+			auto end = chrono::steady_clock::now();
+			chrono::milliseconds duration(chrono::duration_cast<chrono::milliseconds>(end - start));
```

Comment:
Why? It is casted to chrono::milliseconds, because that is the precision I wanted. So the type is clear, and its not so long that I would need a shortcut.